include/verilated.cpp

```diff
@@ -2506,22 +2506,56 @@ VerilatedContext::Serialized::Serialized() {
 
 bool VerilatedContext::assertOn() const VL_MT_SAFE {
     const VerilatedLockGuard lock{m_mutex};
-    return m_s.m_assertOn;
+    return m_s.m_assertOn.any();
 }
```

Comment:
This is a hot function, we should instead use an atomic, or if this is a simple type, not require a lock.